docs/source/configuration/settings-reference.md

```diff
@@ -5,7 +5,7 @@ This is a summary of all the configuration options and what they control.
 ## Main settings
 
 !!! note
-    This list is still incomplete, contributions are welcomed!
+This list is still incomplete, contributions are welcomed!
```

Comment:
Space delete, shouldn't be deleted.

Reply:
I need to disable "Format-on-save" for .md files everytime to make it work ;(